React/central-student-hub/src/UserProfileComponent/Models/OfficeHours.ts

```diff
@@ -0,0 +1,7 @@
+export type OfficeHours = {
```

Comment:
Rename this to be OfficeHour.

Reply:
There are multiple hours in a single instance of this type, so I believe this naming is better!

Reply:
Yes, you are right. I didn't notice.